test/e2e/specs/data-distinct.js

```diff
@@ -0,0 +1,118 @@
+import sandbox from '../helpers/sandbox'
+
+const Backendless = sandbox.Backendless
+
+describe('Data - Distinct', function() {
+  let personTableStore
+
+  let queryBuilder
+
+  sandbox.forSuite()
+
+  function sortArrayWithObjByPropsValue(a, b) {
+    if (a.name > b.name) {
+      return 1
+    }
+    if (a.name < b.name) {
+      return -1
+    }
+
+    return 0
+  }
+
+  const PERSON_TABLE_NAME = 'Person'
+
+  before(async function() {
+    personTableStore = Backendless.Data.of(PERSON_TABLE_NAME)
+
+    await personTableStore.save({ name: 'name1', age: 10 })
+    await personTableStore.save({ name: 'name2', age: 20 })
+    await personTableStore.save({ name: 'name2', age: 20 })
+  })
+
+  beforeEach(async function() {
+    queryBuilder = Backendless.DataQueryBuilder.create()
+  })
+
+  it('load', async () => {
```

Comment:
please give relevant names for each test

Reply:
+